int8 id
float32 setpoint
float32 acceleration